sdk/ml/azure-ai-ml/tests/test_configs/compute/compute-ci-unit.yaml

```diff
@@ -3,6 +3,8 @@ type: computeinstance
 description: some_desc_ci
 
 size: Standard_DS2_v2
+enable_sso: false
```

Comment:
Should we add test cases for when these are not defined and they should default to True?